### ISSUE TITLE
Bent sperm flagella

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9831]
**Preferred term label:**
Bent sperm flagella

**Synonyms**

**Definition (free text, please give PubMed ID)**
For illustrative figure see: WHO laboratory manual for the examination and processing of human semen (Sixth edition)

**Parent term (use hpo.jax.org/app)**
HP:0012868

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399808

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr 
adding as
Bent sperm flagella
HP:0034811
def: The proportion of sperm cells whose tail is not aligned with the midpiece and head (i.e., the flagella is sharply curved or has a sharp angle with respect to the head of the sperm) is above normal limits.
PMID: [20111080](https://pubmed.ncbi.nlm.nih.gov/20111080)

Please reopen this issue of the definition needs to be revised.

**Comment by MagoWyr** [comment]
Yes, that is correct!
________________________________
